### PR TITLE
Color panel

### PR DESCRIPTION
**Pull request by CostircoNazar**

### CODE REVIEW

File: src/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
 import React, {useEffect, useState} from "react";
 import TodoGroup from "../TodoGroup/TodoGroup";
 import TodoSearch from "../TodoSearch/TodoSearch";
+import ColorPanel from "../ColorPanel/ColorPanel";
 import './styles.css'
 import data from "../data/data";
 
+const availableColors = ['yellow', 'red','green','blue'];
```
Comment (HrudakovSerhii) [inline src/Dashboard/Dashboard.js]: Move availableColors inside of ColorPanel component

File: src/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -62,16 +66,35 @@ function Dashboard() {
...
         <div className='dashboard'>
 
+            <div className='dashboardTodoSearch'>
                 <TodoSearch onClear={onClear}
                             onChange={onChange}
                             value={searchValue}
                 />
 
+                <div className='dashboardControlPanel'>
+                    {availableColors.map((color,index)=>
```
Comment (HrudakovSerhii) [inline src/Dashboard/Dashboard.js]: Move map function inside ColorPanel component.

File: src/ColorPanel/ColorPanel.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import './styles.css'
+
+function ColorPanel({setColor,color}){
+
```
Comment (HrudakovSerhii) [inline src/ColorPanel/ColorPanel.js:5]: ColorPanel should accept only setColor function from props.

File: src/Dashboard/styles.css [hunk 4]
```diff
@@ -1,4 +1,17 @@
...
     display: flex;
     flex-flow: column;
+    width: 700px;
+    margin: auto;
+}
+
+.dashboardTodoSearch {
+    display: flex;
+    align-self: center;
+    margin-left: 8%;
```
Comment (HrudakovSerhii) [inline src/Dashboard/styles.css]: Percent margin never works well on dynamic layouts. Use flex-style properties to set position of element

**Review by HrudakovSerhii — COMMENTED** [review]
Please check comments and make required fixes

**Comment by KostyrkoNazar** [comment]
app png

**Comment by CostircoNazar** [comment]
Annotation 2020-05-07 152233

**Pull request merged**